src/EditorFeatures/Core/Microsoft.CodeAnalysis.EditorFeatures.csproj

```diff
@@ -100,7 +88,6 @@
     <InternalsVisibleTo Include="Microsoft.CodeAnalysis.ExternalAccess.FSharp.UnitTests" />
     <!-- Eventually a bunch of these unit tests should move into Roslyn.Services.Implementation.UnitTests
          and this should be removed. -->
-    <RestrictedInternalsVisibleTo Include="Microsoft.CodeAnalysis.TypeScript.EditorFeatures" Key="$(TypeScriptKey)" Partner="VSTypeScript" />
```

Comment:
❔ Is there anything left in the ExternalAccess/VSTypeScript/Api folder for this project after this pull request?

Reply:
there's nothing left

Reply:
Also I had to add a reistricted IVT to `Microsoft.VisualStudio.LanguageServices.TypeScript` in EF.Wpf just in case, because of the unrestricted IVT in EF